### PR TITLE
pull request https://github.com/willdurand/nmap/issues/22

### PR DESCRIPTION
**Pull request by loeken**
timeout can now be passed on as parameter in __construct of Nmap to b…e passed on to the Process executor

### CODE REVIEW

File: src/Nmap/Util/ProcessExecutor.php [hunk 1]
```diff
@@ -22,9 +22,9 @@ class ProcessExecutor
      *
      * @return integer
      */
-    public function execute($command)
+    public function execute($command, $timeout)
     {
-        $process = new Process($command);
+        $process = new Process($command,NULL,NULL,NULL,$timeout);
```
Comment (EmmanuelVella) [inline src/Nmap/Util/ProcessExecutor.php]: You should write `$command, null, null, null, $timeout` to follow CS ;)

**Comment by EmmanuelVella** [comment]
Hi, could you fix the tests so this PR can be merged ? It would be very usefull !

**Comment by EmmanuelVella** [comment]
Fixed by https://github.com/willdurand/nmap/pull/24

**Comment by loeken** [comment]
I can add unit Tests next week currently on holidaysAm 24.07.2017 18:25 schrieb Emmanuel Vella <notifications@github.com>:Hi, could you fix the tests so this PR can be merged ? It would be very usefull !

**Comment by EmmanuelVella** [comment]
@loeken Thank you for your response, but that's not usefull anymore, I created a new PR which was just merged ! Have good holidays ;)

**Comment by loeken** [comment]
Aigh just saw the emails. Even better - less work for me :)Am 26.07.2017 16:58 schrieb Emmanuel Vella <notifications@github.com>:@loeken Thank you for your response, but that's not usefull anymore, I created a new PR which was just merged ! Have good holidays ;)

**Pull request closed without merging**